src/renderer/i18n/de/settings.ts

```
@@ -65,6 +65,15 @@ const settings: SettingMessages = {
   'settings.expert.mayachain.title': 'Mayachain-URLs',
   'settings.expert.midgardMaya.title': 'Midgard Mayachain',
   'settings.expert.mayanodeApi.title': 'MayaNode-API',
+  'settings.expert.evm.title': 'EVM-Ketten RPC-URLs',
+  'settings.expert.evm.eth.title': 'Ethereum RPC',
+  'settings.expert.evm.bsc.title': 'BSC RPC',
+  'settings.expert.evm.arb.title': 'Arbitrum RPC',
+  'settings.expert.evm.avax.title': 'Avalanche RPC',
+  'settings.expert.evm.base.title': 'Base RPC',
+  'settings.evm.rpc.error.url': 'Ungültige EVM RPC URL. Bitte überprüfe diese und versuche es erneut',
+  'settings.evm.rpc.valid': 'Gültige EVM RPC URL',
+  'settings.evm.rpc.unhealthy': 'RPC-Endpunkt antwortet nicht. Überprüfe deine Verbindung oder versuche eine andere URL.',
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)